x/pricefeed/keeper/keeper.go

```diff
@@ -67,6 +68,16 @@ func (k Keeper) SetPrice(
 			index = len(prices) - 1
 		}
 
+		// Emit an event containing the oracle's new price
+		ctx.EventManager().EmitEvent(
+			sdk.NewEvent(
+				types.EventTypeOracleUpdatedPrice,
+				sdk.NewAttribute(types.AttributeMarketID, fmt.Sprintf("%s", marketID)),
+				sdk.NewAttribute(types.AttributeOracle, fmt.Sprintf("%s", oracle.String())),
+				sdk.NewAttribute(types.AttributeMarketPrice, fmt.Sprintf("%s", price.String())),
+				sdk.NewAttribute(types.AttributeExpiry, fmt.Sprintf("%s", expiry.String())),
```

Comment:
I think this will output the time in go's custom text format. I would use some standard like a unix timestamp or https://golang.org/pkg/time/#Time.MarshalText to make it easier for clients to parse

Whichever you use, could you also update `auction`'s events to use the same?

Reply:
https://github.com/Kava-Labs/kava/pull/343/commits/c8e1a85868d6f9f082a3a0753eb9a22049abab9f